src/serialize.js

```diff
@@ -94,7 +94,7 @@ jQuery.fn.extend({
 			var type = this.type;
 
 			// Use .is( ":disabled" ) so that fieldset[disabled] works
-			return this.name && !jQuery( this ).is( ":disabled" ) &&
+			return this.name && !jQuery.find.matchesSelector( this, ":disabled" ) &&
```

Comment:
Input value serialization is not a performance-sensitive area of the codebase... what is the size impact of this change?